### DISCUSSION TITLE
Linux 使用桌面版挂载报错

### DISCUSSION THREAD

**Discussion by LiXiaoQiang123z** [AlistGo/alist#5435] [Q&A]
挂载报错的日志：
start mounting...
<5>NOTICE: webdav root '': --vfs-cache-mode writes or full is recommended for this remote as it can't stream
Fatal error: failed to mount FUSE fs: fusermount: exec: "fusermount3": executable file not found in $PATH
mount finished with code 1 and signal null

我打开先决条件，提示我：
要在Linux系统上运行挂载，需要下载安装：
Nothing
参考: https://github.com/winfsp/cgofuse

我安装cgofuse， 在执行指令：
go install -v ./fuse ./examples/memfs ./examples/passthrough
报错如下：
examples/memfs/memfs.go:21:2: cannot find package "github.com/billziss-gh/cgofuse/examples/shared" in any of:
	/usr/src/github.com/billziss-gh/cgofuse/examples/shared (from $GOROOT)
	/home/lxq/go/src/github.com/billziss-gh/cgofuse/examples/shared (from $GOPATH)
examples/memfs/memfs.go:22:2: cannot find package "github.com/billziss-gh/cgofuse/fuse" in any of:
	/usr/src/github.com/billziss-gh/cgofuse/fuse (from $GOROOT)
	/home/lxq/go/src/github.com/billziss-gh/cgofuse/fuse (from $GOPATH)

**Comment by xhofe** [comment]
debian：`sudo apt install fuse3`
centos：`sudo yum install fuse3`
